### PR TITLE
Disable info in checkpoint logging

### PR DESCRIPTION
**Pull request by sborya**

### CODE REVIEW

File: samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala [hunk 1]
```diff
@@ -204,7 +204,8 @@ class KafkaCheckpointManager(
       val sspToPoll = Collections.singleton(ssp)
       while (currentOffset < newestOffsetLong) {
 
-        val envelopes: java.util.Map[SystemStreamPartition, java.util.List[IncomingMessageEnvelope]] =
+        // will always read a single SSP, but we need to use a map
+        val readSSPs: java.util.Map[SystemStreamPartition, java.util.List[IncomingMessageEnvelope]] =
         try {
           systemConsumer.poll(sspToPoll, POLL_TIMEOUT)
```
Comment (xinyuiscool) [inline samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala]: do a get()

File: samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala [hunk 2]
```diff
@@ -197,32 +194,30 @@ class KafkaCheckpointManager(
...
         }
 
-        val messages: util.List[IncomingMessageEnvelope] = envelopes.get(ssp)
-        val messagesNum = if (messages != null) messages.size else 0
-        info("CheckpointMgr read %s envelopes (%s messages) from ssp %s. Current offset is %s, newest is %s"
-                     format (envelopes.size(), messagesNum, ssp, currentOffset, newestOffset))
-        if (envelopes.isEmpty || messagesNum <= 0) {
-          info("Got empty/null list of messages")
+        debug("CheckpointMgr read %s messages from ssp %s. Current offset is %s, newest is %s"
+                     format (messages.size(), ssp, currentOffset, newestOffset))
```
Comment (prateekm) [inline samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala]: "Read %s messages". 
Fix Indentation (here and line 229)

File: samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala [hunk 3]
```diff
@@ -157,7 +157,7 @@ class TestKafkaCheckpointManager extends KafkaServerTestHarness {
     }
   }
 
-  @Test
+  //@Test
```
Comment (prateekm) [inline samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala]: Why comment out?

File: samza-test/src/test/java/org/apache/samza/test/operator/TestRepartitionJoinWindowApp.java [hunk 4]
```diff
@@ -35,7 +34,7 @@
 public class TestRepartitionJoinWindowApp extends StreamApplicationIntegrationTestHarness {
   private static final String APP_NAME = "UserPageAdClickCounter";
 
-  @Test
+  //@Test
```
Comment (prateekm) [inline samza-test/src/test/java/org/apache/samza/test/operator/TestRepartitionJoinWindowApp.java]: Why comment out?

File: samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala [hunk 5]
```diff
@@ -129,9 +129,6 @@ class KafkaCheckpointManager(
     if (taskNamesToOffsets == null) {
       info("No TaskName to checkpoint mapping provided.  Reading for first time.")
       taskNamesToOffsets = readCheckpointsFromLog()
-    } else {
```
Comment (prateekm) [inline samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala]: Why delete this?

Comment (sborya) [reply samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala]: there is no need for second read.
But for now this change is deleted from this PR.

File: samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala (hunk 2 above)
Comment (sborya) [reply samza-kafka/src/main/scala/org/apache/samza/checkpoint/kafka/KafkaCheckpointManager.scala]: done.

File: samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala (hunk 3 above)
Comment (sborya) [reply samza-kafka/src/test/scala/org/apache/samza/checkpoint/kafka/TestKafkaCheckpointManager.scala]: removed the change.

File: samza-test/src/test/java/org/apache/samza/test/operator/TestRepartitionJoinWindowApp.java (hunk 4 above)
Comment (sborya) [reply samza-test/src/test/java/org/apache/samza/test/operator/TestRepartitionJoinWindowApp.java]: removed from this PR.

**Comment by sborya** [comment]
this functionality was put in as a part of another PR.

**Pull request closed without merging**